web/src/pages/Home/TestsList.tsx

```diff
@@ -98,7 +98,7 @@ const Tests = () => {
         </Pagination>
       </S.Wrapper>
 
-      <CreateTestModal isOpen={isCreateTestOpen} onClose={() => setIsCreateTestOpen(false)} />
```

Comment:
Do we plan to keep this component around?

Reply:
It's going to die. We'll do a big cleanup after we have everything in place